sormas-backend/src/main/java/de/symeda/sormas/backend/common/ConfigFacadeEjb.java

```diff
@@ -534,7 +536,11 @@ public void validateAppUrls() {
 	}
 
 	public int getDashboardMapMarkerLimit() {
-		return getInt(DASHBOARD_MAP_MARKER_LIMIT, -1);
+	    return getInt(DASHBOARD_MAP_MARKER_LIMIT, -1);
+	}
+
+	public boolean getCreateDefaultUsers(){
```

Comment:
boolean getters usually start with "is" instead of "get"

Reply:
can we settle for should?

Reply:
fixed